1-mandatory/1-writers.js

```diff
@@ -61,8 +66,20 @@ Exercise 2:
   "Writer {firstName} {lastName} died at {age} years old."
 */
 
+function getDeadWritersInTheirForties(writer) {
+  return writer.age > 40 && writer.age < 49 && writer.alive === false;
+}
+
+function printSentence(writer) {
+  return console.log(
+    `Writer ${writer.firstName} ${writer.lastName} died at ${writer.age} years old.`
+  );
+}
+
 function logDeadWritersInTheirForties() {
```

Comment:
Praise: Good use of named function here.
Nitpick: The code might be slightly cleaner by using function chaining .e.g
```
writers
  .filter(getDeadWritersInTheirForties)
  .forEach(printSentance)
```